sandbox/user_guide.md

```diff
@@ -59,6 +63,58 @@ f9780d323588f1cdfc3e63e95a8cbdcdd47602ff48a537b51dc5d7834bf466416a716bd4508e904a
 {%- endcapture -%}
 
 {% include copy_snippet.md code=client_secret %}
+</div>
+
+<button class="accordion"> Small ACO (2,500 synthetic beneficiaries) </button>
+<div class="acc_content">
+Client ID:
+{%- capture client_id -%}
+467bb940-7a40-4201-8aee-53d6015362fe
```

Comment:
Were you able to get a token with all these values?

Reply:
changed to the newer values from this pr and tested all of them. Token responses for each one, nice!